app/views/root/_gem_base.html.erb

```diff
@@ -7,14 +7,15 @@
   omit_footer_navigation ||= nil
   omit_footer_border ||= nil
   omit_global_banner ||= nil
-  omit_user_satisfaction_survey ||= nil
   omit_account_navigation ||= nil
   product_name ||= nil
   show_explore_header = show_explore_header === false ? false : true
   show_account_layout ||= false
   account_nav_location ||= nil
   footer_meta ||= nil
   draft_environment ||= ENV["DRAFT_ENVIRONMENT"].present?
+  user_satisfaction_survey ||= nil
```

Comment:
As the card calls for it to be switched off for all circumstances, I'd go a step further than this and suggest removing the both `omit_user_satisfaction_survey` parameter and where it's used on L32 completely.

```suggestion
```

Sidenote - Because `omit_user_satisfaction_survey` is used by other templates[^1], I'd usually be extremely wary of renaming the variable in this manner without changing it where it's used.


[^1]: https://github.com/alphagov/static/blob/3e5bfcea474b4aa54e20c5140248a84f8876a31f/app/views/root/_account.html.erb#L12